testsuite/dockerfiles/server-all-in-one-dev/mirror/product_tree.json

```diff
@@ -15,14 +15,14 @@
         "signed": false,
         "tags": [],
         "update_tag": "SLE-Product-SLES",
-        "url": "https://updates.suse.com/SUSE/Products/SLE-Product-SLES/15-SP4/x86_64/product/"
+        "url": "https://updates.suse.com/SUSE/Products/SLE-Product-SLES/15-SP7/x86_64/product/"
     },
     {
-        "channel_label": "sle-product-sles15-sp4-updates-x86_64",
-        "channel_name": "SLE-Product-SLES15-SP4-Updates for x86_64",
+        "channel_label": "sle-product-sles15-sp7-updates-x86_64",
+        "channel_name": "SLE-Product-SLES15-SP7-Updates for x86_64",
         "gpg_info": [],
         "mandatory": true,
-        "parent_channel_label": "sle-product-sles15-sp4-pool-x86_64",
+        "parent_channel_label": "sle-product-sles15-sp7-pool-x86_64",
```

Comment:
Same, we can't just replace, otherwise the IDs will not match, and if they don't match it might be an issue with suma-toolbox.